### ISSUE TITLE
Favorites are shown twice

### ISSUE THREAD

**Issue by nickoffff** [nextcloud/android#2421]
### Actual behaviour
If I add a file to favorites and later go to the favorites view, it is shown twice.

### Expected behaviour
Show favorite file only once
 
### Steps to reproduce
1. Add file to favorites
2. Go to favorites view

### Environment data
Android version: 8.0.0

Device model: OnePlus 5T

Stock or customized system: Stock

Nextcloud app version: 3.0.3

Nextcloud server version: 13.0.1

**Comment by tobiasKaminsky** [comment]
This is a duplicate of #1732
